client/McFly.js

```diff
@@ -0,0 +1,3 @@
+var McFly = require('mcfly');
```

Comment:
Would like to see `client/mcfly.js` and not having caps in JS file names that we create.